Eshop.WebApi/Controllers/ProductController.cs

```diff
@@ -16,41 +24,46 @@ public class ProductController : Controller
         [HttpGet]
         public List<Product> GetProducts()
         {
-            return Products;
+            return dbContext.Products.ToList();
         }
 
         [HttpGet("{id}")]
         public Product GetProduct(int id)
         {
-            return Products.First(x => x.Id == id);
+            return dbContext.Products.First(x => x.Id == id);
         }
 
         [HttpPost]
         public Product CreateProduct(string title, string description, decimal price, int categoryId)
         {
             var category = CategoryController.Categories.First(x => x.Id == categoryId);
-            var newId = Products.Max(x => x.Id) + 1;
+            var newId = dbContext.Products.Max(x => x.Id) + 1;
             var newProduct = new Product(newId, title, description, price, category);
             
-            Products.Add(newProduct);
+            dbContext.Products.Add(newProduct);
+            dbContext.SaveChanges();
 
             return newProduct;
         }
 
         [HttpDelete("{id}")]
         public void DeleteProduct(int id)
         {
-            var productToBeDeleted = Products.First(x => x.Id == id);
-            Products.Remove(productToBeDeleted);
+            var productToBeDeleted = dbContext.Products.First(x => x.Id == id);
+            dbContext.Products.Remove(productToBeDeleted);
+
+            dbContext.SaveChanges();
```

Comment:
here you have vertical space and in line 44 you don't - good developer is trying to keep the code smell same as much as he can

Reply:
![image](https://github.com/user-attachments/assets/c5806a62-06ed-41e6-af1d-476512da51db)
Im not sure what you mean but those vertical spaces are everywhere same

Reply:
line 37 and 38 has no vertical space between SaveChanges

but for example 47 and 49 are vertically spaced same as 57 and 59